misc/prometheus/mzcompose.py

```diff
@@ -0,0 +1,26 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.mzcompose.services import Service
+
+SERVICES = [
+    Service(
```

Comment:
Should this service definition be put in mzcompose/services.py ? I guess the end goal would be to not have to mess with a given mzcompose workflow to get prometheus up and running. The way we currently have `--redpanda` for several workflows, we should also eventually make it possible to just have a `--prometheus` option to any workflow that will do the required stuff for you.